community/cypher/cypher-compiler-3.3/src/main/scala/org/neo4j/cypher/internal/compiler/v3_3/SeekRange.scala

```diff
@@ -117,14 +122,15 @@ final case class RangeGreaterThan[+V](bounds: Bounds[V]) extends HalfOpenSeekRan
   override def inclusionTest[X >: V](implicit ordering: MinMaxOrdering[X]): Option[X => Boolean] = {
     limit[X].map { bound =>
       val endPoint = bound.endPoint
-      (value: X) => {
-        if (value == null || endPoint == null) {
-          false
-        } else {
-          val cmp = ordering.ordering.compare(value, endPoint)
-          if (bound.isInclusive) cmp >= 0 else cmp > 0
+      (value: X) =>
+        {
+          if (value == null || endPoint == null) {
+            false
+          } else {
```

Comment:
How do we feel about `if-else` constructs without `{}`? I think it can be cleaner sometimes.